src/endpoints/phenotypes.ts

```diff
@@ -1,17 +1,101 @@
-import { idKey } from '../env';
-import { ArrangerProject, searchSqon } from '../sqon/searchSqon';
-import { SetSqon } from './sets/setsTypes';
 import { get } from 'lodash';
+
+import { idKey } from '../env';
+import { ArrangerProject, runQuery } from '../sqon/searchSqon';
 import { replaceSetByIds } from '../sqon/setSqon';
+import { SetSqon } from './sets/setsTypes';
 
 const MAX_PHENOTYPES = 100000;
 
+const extractPsIds = (resp): string[] => (resp?.data?.participant?.hits?.edges || []).map(edge => edge.node[idKey]);
+
+const throwErrorsFromGqlQueryIfExist = resp => {
+    if (resp.errors) {
+        throw new Error(resp.errors.join(','));
+    }
+};
 const getParticipantIds = async (
```

Comment:
Current code does not allow to fetch more than the `max_result_window` limit in ES (most often 10k results)